sklearn/__init__.py

```diff
@@ -17,6 +17,14 @@
 import warnings
 import os
 from contextlib import contextmanager as _contextmanager
+import logging
+
+import six
+
+logger = logging.getLogger(__name__)
+if six.PY2:
+    logger.addHandler(logging.StreamHandler())
+logger.level = logging.INFO
```

Comment:
Maybe add a small comment to state that this ensures that we get a informative message on stdout by default when downloading a dataset on Python 2 (so as to replicate the default behavior of the logging module in Python 3).